src/widget/form/addfriendform.cpp

```diff
@@ -143,6 +142,29 @@ Ignore the proxy and connect to the Internet directly?"), QMessageBox::Yes|QMess
     }
 }
 
+void AddFriendForm::onIdChanged(const QString &id)
+{
+    QString tId = id.trimmed();
+    QRegularExpression dnsIdExpression("^\\S+@\\S+$");
+    bool isValidId = tId.isEmpty() || ToxId::isToxId(tId) || tId.contains(dnsIdExpression);
+
+    QString toxIdText(tr("Tox ID", "Tox ID of the person you're sending a friend request to"));
+    QString toxIdComment(tr("it must have length of 76 hexadecimal characters or similar name@domain.com", "Tox ID format description"));
+
+    if(isValidId)
+    {
+        toxIdLabel.setText(toxIdText + QStringLiteral(" (") + toxIdComment + QStringLiteral(")"));
+    } else
+    {
```

Comment:
This could be more readable, either get rid of the braces or put them all on their own lines.